.travis.yml

```diff
@@ -5,6 +5,10 @@ dist: precise
 matrix:
   fast_finish: true
   include:
+    - php: 7.2
+      env: TYPO3="dev-master as 8.7.0" PHPUNIT=5.7.5
+    - php: 7.2
+      env: TYPO3=^8.7 PHPUNIT=5.7.5
```

Comment:
According to PHPUnits documentation PHPUnit 5 is not compatible to PHP 7.2. I guess to test with PHP 7.2 we should update the testing framework to PHPUnit 6

Reply:
I totally agree (this in fact keeps us from using PHPUnit 6 ATM), but this should be done separately.

Reply:
But as this PHP version isn't supported by the chosen PHPUnit version, I would not allow it

Reply:
PHP 7.2 might not be officially supported by PHPUnit 5.5. However, we don't see any problems using PHPUnit 4.8 for PHP 5.5 up to 7.2. at https://github.com/MyIntervals/emogrifier/blob/master/composer.json . So this might be worth a try.

Reply:
I don't mind if you are using version combination not supported. But I don't want to claim that the testing framework supports that. IMHO if you are knowing what you do, you are able to install the testing-framework with PHP 7.2 already.